cql3/statements/select_statement.cc

```diff
@@ -480,9 +508,10 @@ select_statement::execute_without_checking_exception_message_aggregate_or_paged(
             }
         }();
 
-        co_return shared_ptr<cql_transport::messages::result_message>(
-            ::make_shared<cql_transport::messages::result_message::rows>(result(std::move(generator), std::move(meta)))
-        );
+        auto msg = ::make_shared<cql_transport::messages::result_message::rows>(result(std::move(generator), std::move(meta)));
+        msg->add_tablet_info(tablet_replicas, token_range);
```

Comment:
Can we attach it at higher level? This function doesn't need to know about tablet info.

Reply:
I am not sure, how to do that, this function returns `future<::shared_ptr<cql_transport::messages::result_message>>` and to call `add_tablet_info` at higer level (in `select_statement::do_execute`) I would have to get the value in future but still return future?

Reply:
Done